src/apps/vpn/appconstants.h

```diff
@@ -168,9 +166,18 @@ PRODBETAEXPR(QString, addonBaseUrl,
                  "MZ_ADDON_URL",
                  "https://mozilla-mobile.github.io/mozilla-vpn-client/addons/"))
 
+PRODBETAEXPR(QString, benchmarkDownloadUrl,
+             "https://archive.mozilla.org/pub/vpn/speedtest/50m.data",
+             Constants::envOrDefault(
+                 "MZ_BENCHMARK_DOWNLOAD_URL",
+                 "https://archive.mozilla.org/pub/vpn/speedtest/50m.data"));
+
 PRODBETAEXPR(
-    const char*, benchmarkUploadUrl, "https://benchmark.vpn.mozilla.org/upload",
-    "https://dev.vpn-network-benchmark.nonprod.webservices.mozgcp.net/upload");
+    QString, benchmarkUploadUrl, "https://benchmark.vpn.mozilla.org/upload",
+    Constants::envOrDefault(
+        "MZ_BENCHMARK_UPLOAD_URL",
+        "https://dev.vpn-network-benchmark.nonprod.webservices.mozgcp.net/"
```

Comment:
Question (non-blocking): That reminds me that we also have an additional staging environment https://staging.vpn-network-benchmark.nonprod.webservices.mozgcp.net/ — would this be better to use or maybe it does not really matter?